fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -233,11 +233,11 @@ public void destroy() {
         // this method intentionally left empty
     }
 
-    private FedoraSession session() {
-        if (session == null) {
-            session = sessionFactory.getInternalSession();
+    private FedoraTransaction transaction(final HttpServletRequest servletRequest) {
```

Comment:
Is `servletRequest` needed? it is not used in the method body

Reply:
I will get rid of that.

Reply:
+1